uring/src/main/scala/fs2/io/uring/unsafe/uring.scala

```diff
@@ -22,7 +22,6 @@ import scala.scalanative.unsafe._
 import scala.scalanative.unsigned._
 import scala.scalanative.runtime.Intrinsics._
 
-@link("uring")
```

Comment:
Basically, I don't know how to make the `-luring` this generates go away if a downstream wants to use static linking. I hope someone can tell me its down to my own ignorance 😅